x-pack/plugins/observability_solution/inventory/public/components/search_bar/index.tsx

```diff
@@ -117,9 +79,8 @@ export function SearchBar() {
           appName="Inventory"
           displayStyle="inPage"
           showDatePicker={false}
-          showFilterBar={false}
```

Comment:
There are some visible differences with the filter bar comparing to hosts view: 
| Inventory | Hosts |
| --------- | ------ |
| ![image](https://github.com/user-attachments/assets/c7a9fa07-0cf3-4456-bbba-59f789cb3c7c) | ![image](https://github.com/user-attachments/assets/c60bb599-aa02-4b4f-b44c-72f763cb44fe) |

Should we consider checking the props used [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/search_bar/unified_search_bar.tsx/#L58) maybe or any styles/props applied - I assume that this should be controlled by the search bar so I guess something is missing 🤔

Reply:
I'll take a look, but I won't add any custom CSS... if something is missing/wrong we should open an issue on the Unified Search component.

Reply:
Adding these looks to have solved the problem:
```
          showFilterBar
          showQueryInput
          showQueryMenu
```

🤷🏻‍♂️

Reply:
I checked again and it looks fine now, thanks for fixing that 🙌  (maybe those params add some CSS, idk for sure)